src/action_state.rs

```diff
@@ -16,6 +16,14 @@ use std::marker::PhantomData;
 pub struct ActionData {
     /// Is the action pressed or released?
     pub state: ButtonState,
+    /// The "value" of the binding that triggered the action.
+    ///
+    /// See [`ActionState::action_value()`] for more details.
+    pub value: f32,
+    /// The [`AxisPair`] of the binding that triggered the action.
+    ///
+    /// See [`ActionState::action_axis_pair()`] for more details.
+    pub axis_pair: AxisPair,
```

Comment:
Yeah, now that I think about it, that makes sense. Otherwise there's no way to tell whether the AxisPair value returned is actually meaningful or not.